sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/changestreams/dao/PartitionMetadataDao.java

```diff
@@ -316,7 +329,8 @@ public Timestamp updateToRunning(String partitionToken) {
    */
   public Timestamp updateToFinished(String partitionToken) {
     final TransactionResult<Void> transactionResult =
-        runInTransaction(transaction -> transaction.updateToFinished(partitionToken));
+        runInTransaction(
+            transaction -> transaction.updateToFinished(partitionToken), "updateToFinished");
```

Comment:
We seem to be using tags of the form `xxx=yyy` in some places (e.g. `getPartitionsMatchingState=CREATED`), but `xxx` in other places (like here). Should we normalize?